test/perf/perf_bti_key_translation.cc

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2020-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#include <seastar/testing/perf_tests.hh>
+#include <seastar/testing/test_runner.hh>
+
+#include "schema/schema.hh"
+#include "schema/schema_builder.hh"
+#include "sstables/mx/types.hh"
+#include "sstables/trie/bti_key_translation.hh"
+
+struct lcb_mismatch_test {
+    schema_ptr _s;
+    std::vector<sstables::clustering_info> keys;
+    lcb_mismatch_test() {
+        int n_columns = 16;
+        auto builder = schema_builder("ks", "t")
+            .with_column("pk", int32_type, column_kind::partition_key);
+        for (int i = 0; i <= n_columns; ++i) {
+            builder.with_column(bytes(fmt::format("c{}", i).c_str()), int32_type, column_kind::clustering_key);
+        }
+        _s = builder.build();
+        std::vector<data_value> components;
+        for (int i = 0; i < n_columns - 1; ++i) {
+            components.push_back(data_value(int32_t(0)));
+        }
+        for (int i = 0; i < 100; ++i) {
+            components.push_back(data_value(int32_t(i)));
+            keys.push_back({
+                sstables::clustering_info{
+                    clustering_key_prefix::from_deeply_exploded(*_s, components),
+                    sstables::bound_kind_m::clustering
+                }
+            });
+            components.pop_back();
+        }
```

Comment:
It's better to normalize the results back by dividing by 100 so it will be more understandable. 6k insn/key is nicer than 600k insn/hectokey

Reply:
Can be a follow up, it's just cosmetic.